### DISCUSSION TITLE
How do I get the Galaxy Brain badge?

### DISCUSSION THREAD

**Discussion by Masum-Arifin** [community/community#39067] [Actions] [answered]
**How do I get the Galaxy Brain badge?**

![image](https://user-images.githubusercontent.com/84895237/201628668-3d72c25b-be42-4f00-b32d-c4dcd1b40c68.png)

**Comment by muhammadnurulahsan** [comment, marked as answer]
@Masum-Arifin To achieve the 'Galaxy Brain' achievement, take note of the following:

Answer at least 2 discussions.
Take note, you CAN NOT mark your own answers to your own questions. They have to be drawn by another user.
Discussions should be in a public repository.
There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge.
I hope this answer has helped you, and I all the best in getting your achievement! :)

Reference: https://github.com/orgs/community/discussions/39053

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
